drawing_5.css

```diff
@@ -0,0 +1,108 @@
+.one{
+	height: 300px;
+	width: 100px;
+	float: left;
+}
+.three{
+	height: 300px;
+	width: 100px;
+	float: left;
+}
+.five{
+	height: 300px;
+	width: 100px;
+	float: left;
+}
```

Comment:
Ok, this is called code duplication. Basically, you have the same thing written in many different places, when you don't need to. We apply the `DRY` principle – **Don't Repeat Yourself**. You can DRY up your code by making the above into a single class style. E.g.
    .bigDiv {
        width: 100px;
        height: 300px;
        float: left;
    }